docs/docs/troubleshooting.md

```diff
@@ -12,17 +12,17 @@ Please check the [list of frequently asked questions](faq.md).
 
 * Surround Sound isn't supported in xemu yet, you should disable this in your [EEPROM](eeprom.md) if you have enabled it.
 
-## Flatpak release will not load BIOS / MCPX ROM / Hard Drive
+## Flatpak release will not load the Hard Drive
```

Comment:
Does this section not apply to BIOS/MCPX ROM images if they are placed outside the sandbox?

Reply:
It looks like the flatpak now has full filesystem read permissions, so only files xemu needs to write should be affected (e.g. hard disk image, EEPROM, screenshots).

Reply:
Correct, I evaluated that earlier today.